readme.md

```diff
@@ -742,6 +742,7 @@
 - [GDPR](https://github.com/bakke92/awesome-gdpr#readme) - Regulation on data protection and privacy for all individuals within EU.
 - [DevSecOps](https://github.com/TaptuIT/awesome-devsecops#readme) - Integration of security practices into [DevOps](https://en.wikipedia.org/wiki/DevOps).
 - [Executable Packing](https://github.com/dhondta/awesome-executable-packing#readme) - Packing and unpacking executable formats.
+- [Apple Security](https://github.com/BlackSquirrelz/awesome-apple-security#readme) - Apple macOS, iOS related cyber security tools, techniques and resources.
```

Comment:
The description should not describe what the list contains. It should describe the topic.

Reply:
Changed to: _Apple Security resources._